code/modules/mob/living/silicon/robot/robot_laws.dm

```diff
@@ -66,7 +66,7 @@
 	laws.set_zeroth_law("Do not state this law. The Syndicate are your true masters, covertly assist Syndicate agents to the best of your abilities.")
 
 /mob/living/silicon/robot/set_zeroth_law()
-	if(mmi.syndiemmi)
+	if(mmi && mmi.syndiemmi)
```

Comment:
```suggestion
	if(mmi?.syndiemmi)
```

Reply:
applied but if an MMI gets a value of 0 it will still pass
this will likely just cause runtimes and nothing else however it will be very funny